app/models/user.rb

```diff
@@ -140,6 +145,8 @@ class User < ActiveRecord::Base
   validate :unique_email, if: ->(user) { user.email_changed? }
   validate :owns_notification_email, if: ->(user) { user.notification_email_changed? }
   validates :avatar, file_size: { maximum: 200.kilobytes.to_i }
+  validates :longest_streak, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
Since we don't store this information by a user action, I think we can drop these 2 validations.